data/json/effects.json

```
@@ -1880,6 +1881,25 @@
     "base_mods": { "dex_mod": [ -4, 0 ] },
     "blood_analysis_description": "Clostridium Tetani Infection"
   },
+  {
+    "type": "effect_type",
+    "id": "rat_bite_fever",
+    "name": [ "puckered scratches" ],
+    "apply_message": "Your skin starts burning around some of your scratches.  Some aspirin might relieve the effects.",
+    "remove_message": "The fever breaks.",
+    "rating": "bad",
+    "resist_effects": [ "aspirin" ],
+    "base_mods": {
+      "stamina_min": [ -50 ],
+      "stamina_max": [ -100 ],
+      "stamina_chance": [ 300 ],
+      "base_mods": { "int_mod": [ -1, 0 ] },
```

Looks like a merge error perhaps? This change makes it valid, but double check that it matches your intent.
```suggestion
      "int_mod": [ -1, 0 ],
```